ecosystem/platform/server/app/javascript/controllers/table_row_controller.ts

```diff
@@ -0,0 +1,13 @@
+import { Controller } from "./controller";
+
+export default class extends Controller<HTMLTableRowElement> {
+
+  tableRowClick(event: Event) {
+    if (!(event.currentTarget instanceof Element)) return;
+    const validatorAddress = event.currentTarget.querySelector('a');
```

Comment:
I would change this to exit early if a `<a>` isn't found or if event.target is the anchor. Then I would change it to not call event.preventDefault() (it shouldn't be necessary if event.target is not the anchor). I would also exit early if `event.isDefaultPrevented()` (this will make it possible to have other clickable things in the table, like buttons which could preventDefault() in their click handler). I would also rename the variable to `anchor` since this is intended to be a generic table row component (i.e. the link could be anything, not just a validator address).